src/Products/PloneMeeting/migrations/migrate_to_4217_1.py

```
@@ -33,21 +33,43 @@ def _configureCssTransforms(self):
+                gic_notes_config = [row for row in default_value if row['name'] == "groupsInChargeNotes"]
+                if gic_notes_config:
+                    gic_notes_config = gic_notes_config[0]
+                    item_fields_config += (gic_notes_config, )
+                    cfg.setItemFieldsConfig(item_fields_config)
+        # update new fields groupsInChargeNotes on items
+        self.initNewHTMLFields(
+            query={'meta_type': ('MeetingItem')},
+            field_names=('groupsInChargeNotes', ))
+        logger.info('Done.')
+
     def run(self, extra_omitted=[], from_migration_to_4200=False):
```

Ruff (0.15.11)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)